decentraland/unity-renderer
# fix: fail stream includes exception & better error handling of some assets during loading screen

## What does this PR change?

Reports a critical error (bypassing standard error flow) when any gltf, ab, wearable, avatar shape, renderables throws a load error while in loading screen. A popup is shown immediatelly on desktop client.

The loading screen popup implementation is at [explorer-desktop](https://github.com/decentraland/explorer-desktop) project

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=fix/init-freeze
2. Restart the browser many times and wait until the loading finishes
3. Run around the world and check objects are being loaded correctly

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
